tests/foreman/cli/test_container_management.py

```diff
@@ -535,8 +536,9 @@ def _finalize():
         assert lib_path not in finds
         assert repo_path not in finds
         assert cv_path in finds
-        paths = [f.strip() for f in finds.split('\n') if 'NAME' not in f and len(f)]
-        assert len(paths) == 1
+        if not is_open('SAT-39878'):
+            paths = [f.strip() for f in finds.split('\n') if 'NAME' not in f and len(f)]
+            assert len(paths) == 1
 
         # 4. Try podman search/pull for Library images, ensure it fails.
```

Comment:
The library images tested here are authenticated? Asking since the bug is around unauthenticated repositories.

Reply:
(specifically part 4)

Reply:
Those tested here are authenticated (the LCE default).

However, two of the tests above set `registry-unauthenticated-pull` to `true` for their LCEs and those leak to our case.

https://github.com/SatelliteQE/robottelo/blob/bb777ed158af098394412ceb8d3328ec74c537a5/tests/foreman/cli/test_container_management.py#L210-L217
https://github.com/SatelliteQE/robottelo/blob/bb777ed158af098394412ceb8d3328ec74c537a5/tests/foreman/cli/test_container_management.py#L323-L330